skf/markdown/code_examples/java/31-code_example--Single_input_validation_control--.md

```diff
@@ -0,0 +1,131 @@
+Single input validation control 
+--------------------------------
+
+***Example:***
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+import java.time.LocalDateTime;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class InputValidation {
+	
+	private AuditLog Log = new AuditLog(); 
+	public String validate = "pass"; 
```

Comment:
This property should be private since it's not used in other classes, and even if it is used on other places, there should be a public getter for this field and keep it as private. Eg:

```java

private String validate = "pass"

public String getValidate() {
    return this.validate;
}
```